arrays_lab.js

```diff
@@ -0,0 +1,299 @@
+if (firstArr.length > secondArr.length) {
+  console.log(firstArr);
+} else if (firstArr.length < secondArr.length){
+  console.log(secondArr);
+} else {
+  console.log("They are the same size")
+}
+
+//console.log(----------------------------------------------------------------)
+//10)
+//Given an array, log its middle element.
+//If the middle of the array is between two elements, log both of them.
+
+//let hasExactMiddleArr = [1,5,3] -// log 5
+
+//let hasNoExactMiddleArr = [1,4,6,9]-// log 4,6
+
+let numArray = [2,3,4,5,6];
+let midArr = Math.floor(numArray.length -1 / 2);
+if (midArr % 2 === 0) {
+  console.log(midArr)
+  //result.push (i)
+}
+//console.log(------------------------------------------------------------------)
+//11. Write code that logs every value in an array using a for loop
+
+let logMeOutFor = ["I", "am", "a", "happy", "array"]
+for (let i = 0; i<logMeOutFor.length; i++){
+  console.log(logMeOutFor[i]);
+}
+
+
+//console.log(------------------------------------------------------------------)
+//12. Write code that logs every value in an array using a while loop//
+//during a condition is correct do this . While something is something do this.
+//There is no i in while loop
+let logMeOutFor = ["I", "am", "a", "happy", "array"]
+let num = 0
+while (num <logMeOutFor.length){
+console.log(logMeOutFor[num])
+num++;
+}
+
+//console.log(------------------------------------------------------------------)
+
+//13. Write code that logs every value in an array using a for/of loop
+//for of goes though every item that is inside of the array
+
+let logMeOutFor = ["I", "am", "a", "happy", "array"]
+for (let item of logMeOutFor) }
+  console.log(item);
+{
+
+
+//console.log(------------------------------------------------------------------)
+
+// //14. Given an array of numbers, write code that logs the sum of all the numbers
+//
+let total=0
+let sumArray =[1, 2, 3, 4, 5, 6];
+for (let i = 0; i < sumArray.length; i++){
+  console.log("total: ", total)
+  console.log("array value", sumArray[i])
+      total+= sumArray[i]
+}
+console.log(total)
+
+//
+//
+//
+// // let sumArray = [1,2,3,4,5,6];
+// // for(let i =0, sum = 0; i<sumArray.length; sum +=sumArray[i++];
+// // //intializer, condition and increment
+// //
+// // maybe :
+// // var arr = [1,2,3,4];
+// // var total=0;
+// // for(var i in arr) { total += arr[i]; }
+// //
+// //
+//
+// //console.log(----------------------------------------------------------------)
+// //15. Given an array of numbers, write code that logs all the odd numbers
+//
+let evenAndOddArr = [1,5,2,4,11,12,99,100]
+for (let i=1; i>evenAndOddArr;i++) {
+  if (evenAndOddArr %2 !== 0){
+    console.log(i);
+  }
+}
+//
+//
+// //console.log(--------------------------------------------------------------)
+//
+// //16. Given an array of numbers, write code that logs the sum of all the even numbers
+//
+//
+let evenAndOddArr = [1,5,2,4,11,12,99,100]
+for (let i=2; i>evenAndOddArr;i++) {
```

Comment:
Same problem as above. Loop is not starting from 0 which means you are skipping over first two items. Look at my comment above.